arcgis-ios-sdk-samples/Display information/Filter by time extent/FilterByTimeExtentViewController.swift

```diff
@@ -0,0 +1,106 @@
+// Copyright 2021 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import ArcGIS
+import ArcGISToolkit
+
+class FilterByTimeExtentViewController: UIViewController {
+    /// The map view.
+    @IBOutlet var mapView: AGSMapView! {
+        didSet {
+            let map = AGSMap(basemapStyle: .arcGISTopographic)
+            map.operationalLayers.add(featureLayer)
+            mapView.map = map
+            // Set the map view's viewpoint.
+            let center = AGSPoint(x: -58.495293, y: 29.979774, spatialReference: .wgs84())
+            mapView.setViewpoint(AGSViewpoint(center: center, scale: 1.5e8))
+        }
+    }
+    
+    /// The date formatter.
+    static let formatter: DateFormatter = {
+        let formatter = DateFormatter()
+        formatter.dateFormat = "yyyy/MM/dd HH:mm"
+        return formatter
+    }()
+    
+    /// The default date and time for the starting and ending thumb.
+    let currentTimeExtent = AGSTimeExtent(
+        startTime: formatter.date(from: "2005/10/01 05:00")!,
+        endTime: formatter.date(from: "2005/10/31 05:00")!
+    )
+    
+    /// The feature layer tracking hurricanes in 2005.
+    let featureLayer = AGSFeatureLayer(
+        item: AGSPortalItem(
+            portal: .arcGISOnline(withLoginRequired: false),
+            itemID: "49925d814d7e40fb8fa64864ef62d55e"
+        ),
+        layerID: 0
+    )
+    
+    /// The time slider from the ArcGIS toolkit.
+    let timeSlider = TimeSlider()
```

Comment:
I was surprised to see this created here and not in the storyboard, but I know why now. It would be nice if it implemented [`init?(coder:)`](https://github.com/Esri/arcgis-runtime-toolkit-ios/blob/de6ebdb144097681b40d1d12ffe61e7985aadaac/Sources/ArcGISToolkit/TimeSlider.swift#L707-L709) so that it could be used from a storyboard.

Reply:
Hm not sure what you mean here. Can you explain further?

Reply:
I was referring to `TimeSlider` in the toolkit. Nothing for you to do here.